### PR TITLE
Updating documentation links

### PR DESCRIPTION
**Pull request by mohammedeisa**
This commit contains updates to the sample options page documentation links.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/dev.md [hunk 1]
```diff
@@ -16,14 +16,14 @@ Are you a user reporting a bug with a theme or plugin?
...
 
-If you have purchased a theme or plugin that utilizes Redux, please contact the author for support. If the author is unavailable for support or has discontinued support, please refer to this before posting to our issue tracker: https://docs.redux.io/core/support-defined/  If you are unable to get support for the author, we might be able to help you with a premium support purchase (https://redux.io/extension/premium-support).
+If you have purchased a theme or plugin that utilizes Redux, please contact the author for support. If the author is unavailable for support or has discontinued support, please refer to this before posting to our issue tracker: https://devs.redux.io/core/support-defined/  If you are unable to get support for the author, we might be able to help you with a premium support purchase (https://redux.io/extension/premium-support).
```
Comment (KevinBatdorf) [inline .github/ISSUE_TEMPLATE/dev.md:19]: https://devs.redux.io/core/support-defined gives a 404

File: .github/ISSUE_TEMPLATE/dev.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Think you found a bug?
 ======================
 Please use the "BUG REPORT" template below. Remember, we are rabid about backwards compatability. Do not submit fixes that break previous ways of doing so or they will be rejected.
 
-Also, unless you provide a Support Hash, your request will also be rejected.  NO exceptions.  If you do not including this information and we close your ticket, please do not ask why. See See https://docs.redux.io/core/generating-a-support-hash/ for more details.
+Also, unless you provide a Support Hash, your request will also be rejected.  NO exceptions.  If you do not including this information and we close your ticket, please do not ask why. See See https://devs.redux.io/core/generating-a-support-hash/ for more details.
```
Comment (KevinBatdorf) [inline .github/ISSUE_TEMPLATE/dev.md:39]: 404

File: sample/barebones-config.php [hunk 3]
```diff
@@ -308,7 +308,7 @@
 
 $section = array(
 	'title'      => esc_html__( 'Text Area', 'your-textdomain-here' ),
-	'desc'       => esc_html__( 'For full documentation on this field, visit: ', 'your-textdomain-here' ) . '<a href="//docs.redux.io/core/fields/textarea/" target="_blank">//docs.redux.io/core/fields/textarea/</a>',
+	'desc'       => esc_html__( 'For full documentation on this field, visit: ', 'your-textdomain-here' ) . '<a href="https://devs.redux.io/core-fields/textarea.html" target="_blank">//docs.redux.io/core-fields/textarea.html</a>',
```
Comment (KevinBatdorf) [inline sample/barebones-config.php:311]: This one still says docs

**Review by KevinBatdorf — CHANGES_REQUESTED** [review]
I only looked at a few of them but the pages don't exist. Is there a plan to add these pages before this is updated?

(The old pages don't seem to exist either)

**Comment by mohammedeisa** [comment]
@KevinBatdorf I think you can merge this commit as is and I will fix the undefined and the docs.redux.io ones.
About the inexistent pages, I prefer to keep them until we create pages with the same usage.

**Comment by KevinBatdorf** [comment]
Can you create the pages first? Or otherwise add redirects?

Also, I'm not familiar enough with the codebase to be merging.

**Comment by mohammedeisa** [comment]
> Can you create the pages first? Or otherwise add redirects?
> 
> Also, I'm not familiar enough with the codebase to be merging.

Currently, creating new pages for documentation is not a priority.
The priority is to make the documentation links in the plugin work.

File: sample/barebones-config.php (hunk 3 above)
Comment (KevinBatdorf) [reply sample/barebones-config.php:311]: Screen Shot 2021-01-22 at 8 42 57 AM

**Comment by KevinBatdorf** [comment]
> The priority is to make the documentation links in the plugin work.

In general we should never add anything broken to production code. Are all the URLs working now?

**Comment by mohammedeisa** [comment]
@KevinBatdorf It seems that there is something wrong with my IDE.
I found many other docs.redux.io occurrences that replaced them.
I commented out the unexisting documentation links and added links to the old documentation website docsv3.redux.io for the links that don't exist in devs.redux.io temporary until we create the same pages in devs.redux.io

**Pull request merged**